onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,120 @@
+"""
+Delete users management command.
+"""
+import sys
+from django.contrib.auth.models import User
+from onadata.apps.logger.models import XForm
+from onadata.apps.logger.models import Instance
+from onadata.apps.logger.models import Project
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+
+def get_user_object_stats(
+        username):  # pylint: disable=R0201
+    """
+    Get User information.
+    """
+    # Get the number of projects for this user
+    user_projects = Project.objects.filter(
+        created_by__username=username).count()
+    # Get the number of forms
+    user_forms = XForm.objects.filter(
+        user__username=username).count()
+    # Get the number of submissions
+    user_sumbissions = Instance.objects.filter(
+        user__username=username).count()
+
+    user_response = input(
+        "User account '{}' has {} projects, "
+        "{} forms and {} submissions. "
+        "Do you wish to continue "
+        "deleting this account?".format(
+            username,
+            user_projects,
+            user_forms,
+            user_sumbissions
+            ))
+
+    return user_response
+
+
+def inactivate_users(users, user_input):
+    """
+    Soft deletes the user termporarily.
+    """
+    if users:
+        for user in users:
+            username, email = user.split(':')
+            user_response = 'True'
+            if user_input == 'False':
+                # If the --user_input flag is not provided.
+                # Get acknowledgement from the user on this
+                user_response = get_user_object_stats(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many positional arguments for function call](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=4523955)